File: app/src/main/java/com/example/contacts/database/dao/LogUserDao.java
package com.example.contacts.database.dao;

import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.Query;

import com.example.contacts.database.entity.LogUser;

@Dao
public interface LogUserDao {
    @Query("SELECT * FROM loguser LIMIT 1")
    LogUser getLogUser();

    @Insert
    void insertAll(LogUser users);

    @Query("DELETE FROM loguser WHERE login = :login")
    void deleteAll(String login);
}

File: app/src/main/java/com/example/contacts/database/dao/LogUserDao.kt
package com.example.contacts.database.dao

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.Query
import com.example.contacts.database.entity.LogUser

@Dao
interface LogUserDao {
    @get:Query("SELECT * FROM loguser LIMIT 1")
    val logUser: LogUser?

    @Insert
    fun insertAll(users: LogUser?)

    @Query("DELETE FROM loguser WHERE login = :login")
    fun deleteAll(login: String?)
}